mavros_msgs/msg/ESCTelemetryItem.msg

```diff
@@ -0,0 +1,6 @@
+uint8 temperature
```

Comment:
Need to add Header and fillup receiving time if message do not have it.

Thus way you could check what items been updated.